spec/jstests-spec.js

```diff
@@ -305,41 +304,145 @@ describe("Check correct url params", function() {
   })
 });
 
-});
-
 describe("Node search", function() {
-  const cy = cytoscape({ 
+  const cy = cytoscape({
     elements: [
-    { data: { id: "A" } },
-    { data: { id: "B" } },
-    {
-      data: {
-        id: "AB",
-        source: "A",
-        target: "B"
-      }
-    }]
+      { data: { id: "A" } },
+      { data: { id: "B" } },
+      {
+        data: {
+          id: "edgeAB",
+          source: "A",
+          target: "B"
+        }
+      }]
+  });
+
+  let numSelected;
+  let nodeError;
+  let query;
+  beforeEach(function() {
+    document.body.innerHTML = "";
+
+    numSelected = document.createElement("label");
+    numSelected.id = "num-selected";
+    document.body.appendChild(numSelected);
+
+    nodeError = document.createElement("label");
+    nodeError.id = "node-error";
+    document.body.appendChild(nodeError);
+
+    query = document.createElement("input");
+    query.id = "node-search";
+    document.body.appendChild(query);
+  });
+
+  it("should be a successful node search", function() {
+    query.value = "A";
+    const result = search(cy, "node", searchNode);
+
+    // should not display error message
+    expect(nodeError.innerText).toBe("");
+    expect(result.id()).toBe("A");
+  });
+
+  it("should be an unsuccessful node search", function() {
+    query.value = "C";
+    const result = search(cy, "node", searchNode);
+
```

Comment:
Since you have the result (target) here, could you also assert that the node is correctly highlighted? e.g. `node.numericStyle('border-width')`